tests/rocker_scripts/matrix.json

```diff
@@ -17,7 +17,8 @@
     "install_python.sh",
     "install_jupyter.sh",
     "install_julia.sh",
-    "install_cuda-10.1.sh"
+    "install_cuda-10.1.sh",
+    "install_texlive.sh"
```

Comment:
Is this necessary?
Isn't it tested via `install_verse.sh`?

Reply:
it is.  you said: 

> You should add the test case install_texlive.sh on rocker/cuda:latest to this file.

I'm not sure I entirely understand the syntax of matrix.json.  as you see below, I added this check using `install_texlive.sh`, and thought I had to list that here.

Reply:
@eitsupi ok, I've reverted this list back to what it was before.  I've kept the addition of adding a test on `rocker/cuda:latest`, for consistency with other tests I've just had it test against `install_verse.sh` which as you say will also test `install_texlive.sh` anyway.  Let me know what you think.